Makefile

```diff
@@ -4,7 +4,7 @@
 install:
 	pip install -U setuptools pip
 	pip install -e .
-	pip install -r tests/requirements.txt
+	pip install -r requirements-dev.txt
```

Comment:
Why did you move this? Is it used in things other than tests?

Reply:
Includes mypy etc. too. Just making it more consistent with other repos.

Reply:
I guess that's okay for now, then. Maybe someday this project will be ready to make use of more precise per-env lockfiles (similar to what I'm trying to achieve with https://github.com/cherrypy/cheroot/tree/e1a45fa/requirements).